HISTORY.rst

```diff
@@ -14,6 +14,17 @@ Release History
 
 * Updated both controlplane and dataplane SDKs to now use the newer 2021-06-30-preview API version.
 
+**IoT Central updates**
+
+* Added commands for Edge devices and modules:
+  - az iot central device edge list
```

Comment:
edge devices are not the same as normal devices or direct subgroups of normal devices. It may make more sense to separate this command group as az iot central edge list/get/module/etc. 

For example, we separate `az iot device` and `az iot edge`
![image](https://user-images.githubusercontent.com/73560279/155217817-4ad283f4-29fa-4b73-b097-edb617ecc67d.png)


Also list is listed twice - typo?

Reply:
To add on - how are we expecting users to create/delete/update edge devices using the CLI?

Reply:
For IoT Hub commands, device identity operations work mostly the same across edge & non-edge devices. The difference between creating an edge device vs non-edge devices is including the `--edge-enabled` flag.

Edge deployments and setting modules on a particular edge device are separated in the `az iot edge` command group.

Reply:
this was a result of an internal discussion. In IoT Central, Edge devices are just a subset of "devices". They also share the same commands like "show-credentials" or "registration-info".
Also in IoT Central a device is just an "entry" until a template is assigned to it, so edge devices can be created as usual with "device create" by passing an edge template.

Reply:
About manifests, in Central these are per template and can only be changed from device-template command group.

Reply:
may help to add to the help "to create an edge device, use az iot central device --edge-enabled" so users can quickly see how to add devices

Reply:
Regarding the edge vs non edge devices - is there a need to separate the show and list commands if the information would be the same (for an edge device). For example, in the screenshot below the information for the simulated edge device is the same as the information received from just using the show command without the edge. For list, we can just have an extra "only show edge devices" flag to the az iot central device list. (ex: `az iot central device list --edge -n {name _of_the_app}`) Thus, without show and list, the subgroup edge would only contain commands that only edge devices have.

Here is my testing, the first device1 is a regular device, the second 1060l0u3ktk is a non simulated edge device and the third 1ihf8g7bqo0 is a simulated edge device.
![image](https://user-images.githubusercontent.com/73560279/155413972-813d98b6-63de-42b5-be71-661473c843e3.png)

Reply:
Fixed this by following suggestions. Edge group will only have subgroups.